docs/tutorial.rst

```diff
@@ -56,7 +58,11 @@ For example::
     if __name__=="__main__":   
         options = Job.Runner.getDefaultOptions("./toilWorkflowRun")
         options.logLevel = "INFO"
-        Job.Runner.startToil(HelloWorld("woot"), options)
+        job = HelloWorld("woot")
+        Job.Runner.startToil(job, options)
+
+        with Toil(options) as toil:
+            toil.run(job)
     
```

Comment:
startToil() and the Toil CM are two mutually exclusive. You can't combine them in the same application.